packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -412,6 +539,72 @@ class _CupertinoAlertActionSection extends StatelessWidget {
 
   bool get _layoutActionsVertically => children.length > 2;
 
+  Widget _buildHorizontalDivider() {
+    return new Container(
+      height: 0.3,
```

Comment:
This will only work on 3x density devices right? Might have to read the MediaQuery